lib/Interface/Container.cpp

```diff
@@ -277,4 +279,36 @@ template <typename T, size_t N> T *MemRef<T, N>::release() {
   return temp;
 }
 
+template <typename T, size_t N>
```

Comment:
What is the purpose of this overload? Is it added **only** for the purpose of checking two graphs in tests?

Reply:
yes.